src/user/header/component.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import { Grid, Paper } from '@material-ui/core'
+import PropTypes from 'prop-types'
```

Comment:
Let's rearrange these imports to respect the order we've established.

Reply:
Done